development/test-log-collector/cmd/app/mainerr_test.go

```diff
@@ -0,0 +1,336 @@
+package app
+
+import (
+	"io"
+	"io/ioutil"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+
+	octopusTypes "github.com/kyma-project/test-infra/development/test-log-collector/pkg/resources/clustertestsuite/types"
+)
+
+func Test_getNewestClusterTestSuite(t *testing.T) {
+
+	date2009 := metav1.NewTime(time.Date(2009, 11, 17, 20, 34, 58, 651387237, time.UTC))
+	date2020 := metav1.NewTime(time.Date(2020, 11, 17, 20, 34, 58, 651387237, time.UTC))
```

Comment:
Maybe change data to minutes, or hours, not years.

Reply:
Why would I? The only thing that matters is the fact that one is earlier than another

Reply:
Yeah, but you know, difference will be by minutes, or hours, not years. This is only for test time which will appear more often than years 😄

Reply:
This is not a big problem for me.